README.md

```diff
@@ -1,3 +1,4 @@
 # TicTacToe_Ruby_v2
 
-[![Build Status](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2.svg?branch=master)](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2)
+
+[![Build Status](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2.svg?branch=convert_to_gem)](https://travis-ci.org/td-extreme/TicTacToe_Ruby_v2)
```

Comment:
So I guess I just would never change this when I'm working on a feature branch. I would be checking CI directly to see what's going on.